extensions/vscode/src/extension.ts

```diff
@@ -12,8 +12,11 @@ import { newRoute } from "./commands";
 export function activate(
   context: vscode.ExtensionContext
 ): vscode.ExtensionContext {
+  installCLI();
```

Comment:
I think we should prompt the user to install dart frog CLI.

Reply:
I agree, however the extension at this point will be unable to do absolutely nothing if the CLI is not installed. Are we sure we want to have this as an option?

Reply:
If this feature is required, I'll be tackling it in another Pull Request.